src/Core/Domain/CustomerService/QueryHandler/GetCustomerServiceSignatureHandler.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\CustomerService\QueryHandler;
+
+use PrestaShop\PrestaShop\Core\ConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Domain\CustomerService\Query\GetCustomerServiceSignature;
+
+/**
+ * @internal
+ */
+final class GetCustomerServiceSignatureHandler implements GetCustomerServiceSignatureHandlerInterface
+{
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
+    /**
+     * @param ConfigurationInterface $configuration
+     */
+    public function __construct(ConfigurationInterface $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(GetCustomerServiceSignature $query)
+    {
+        $signature = $this->configuration->get('PS_CUSTOMER_SERVICE_SIGNATURE');
+
+        return str_replace(
+            '\r\n', '\\n',
```

Comment:
o_O looks not nice to me
@PierreRambaud any opinion on this ?

Reply:
Look really weird.
Please investigate, and remove this ugly code if possible.

Reply:
It comes from legacy, it seems that it used to work so I'll keep it, unless you have some suggestions for improvement.

Reply:
We actually dont understand the need for this :/ it's going to replace *real* characters \r\n, not the ASCII version which has a semantic meaning

It seems useless. Let's remove it and see if QA finds an issue, it really really seems useless

Reply:
Done.